### DISCUSSION TITLE
Replica number does not update

### DISCUSSION THREAD

**Discussion by asrachid** [Security-Onion-Solutions/securityonion#12832] [2.4] [answered]
### Version

2.4.50

### Installation Method

Security Onion ISO image

### Description

other (please provide detail below)

### Installation Type

Distributed

### Location

on-prem with Internet access

### Hardware Specs

Meets minimum requirements

### CPU

4

### RAM

16

### Storage for /

200

### Storage for /nsm

200

### Network Traffic Collection

tap

### Network Traffic Speeds

1Gbps to 10Gbps

### Status

Yes, all services on all nodes are running OK

### Salt Status

No, there are no failures

### Logs

No, there are no additional clues

### Detail

I update the replica number for an index that has only one primary shard (by going to Administration –> Configuration –> elasticsearch –> index_settings –> so-INDEX-NAME –> index_template –> template –> settings –> index –> number_of_replicas ), but when I launch this cmd (so-elasticsearch-shards-list) in the manager, I still have just one primary shard and my data is not replicated to my second search node. Is this what we should do to update the replica number or is there something else?

### Guidelines

- [X] I have read the discussion guidelines at https://github.com/Security-Onion-Solutions/securityonion/discussions/1720 and assert that I have followed the guidelines.

**Comment by reyesj2** [comment, marked as answer]
If you've made the change it sounds like you're just going to be waiting for the index to rollover and then you will see the changes reflected in all subsequent indices
